app/controllers/application_controller.rb

```diff
@@ -1,24 +1,2 @@
 class ApplicationController < ActionController::Base
```

Comment:
- [ ] Please configure the first user to be the session's user in this file. You can achieve this by creating a function `current_user` which returns the first user with `User.first`.